File: ProviderTest/app/src/main/java/webinstats/ProviderTest/MainActivity.java
package webinstats.ProviderTest;

import android.support.v7.app.AppCompatActivity;
import android.os.Bundle;

import java.util.HashMap;
import java.util.Map;

import webinstats.android_wis.Webinstats;

public class MainActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
        Webinstats wiso = new Webinstats("//wis.webinstats.com","1549","0");
        Map<String,String> map = new HashMap<>();
        map.put("p","page name");
        map.put("_enable_push","1");
        wiso.execute(this,map);
    }
}

File: MyApplication3/app/src/main/java/com/webinstats/myapplication/MainActivity.kt
package com.webinstats.myapplication

import android.app.PendingIntent.getActivity
import android.support.v7.app.AppCompatActivity
import android.os.Bundle
import webinstats.android_wis.Webinstats

class MainActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)
        val wiso = Webinstats("//_YOUR_DOMAIN_.webinstats.com/","__YOUR_COMPANY_ID_","0")
        val map = HashMap<String,String>()
        map.put("p","Other")
// For each activity => call execute function on onCreate() method
        wiso.execute(this,map)

    }
}
